src/Sulu/Bundle/ContentBundle/Content/Types/InternalLinks.php

```diff
@@ -78,9 +77,11 @@ public function read(
         $languageCode,
         $segmentKey
     ) {
-        $data = $node->getPropertyValueWithDefault($property->getName(), array());
-
-        $this->setData($data, $property, $webspaceKey, $languageCode);
+        $data = array();
+        if ($node->hasProperty($property->getName())) {
+            $data = $node->getProperty($property->getName())->getString();
+        }
```

Comment:
@dantleech Why did you have to change this? How is this if different from the `getPropertyValueWithDefault`?